spec/integ/crypto/crypto.spec.ts

```diff
@@ -2324,27 +2375,18 @@ describe.each(Object.entries(CRYPTO_BACKENDS))("crypto (%s)", (backend: string,
          * https://spec.matrix.org/v1.6/client-server-api/#get_matrixclientv3sync
          * @param secretStorageKey
          */
-        function sendSyncResponse(secretStorageKey: string) {
-            syncResponder.sendOrQueueSyncResponse({
-                next_batch: 1,
-                account_data: {
-                    events: [
-                        {
-                            type: "m.secret_storage.default_key",
-                            content: {
-                                key: secretStorageKey,
-                            },
-                        },
-                        // Needed for secretStorage.getKey or secretStorage.hasKey
-                        {
-                            type: `m.secret_storage.key.${secretStorageKey}`,
-                            content: {
-                                algorithm: SECRET_STORAGE_ALGORITHM_V1_AES,
-                            },
-                        },
-                    ],
-                },
-            });
+        function sendSyncResponseWithUpdatedAccountData() {
+            try {
+                syncResponder.sendOrQueueSyncResponse({
+                    next_batch: 1,
+                    account_data: {
+                        events: accountDataEvents,
+                    },
+                });
+            } catch (err) {
+                // It's ok if it fails here, the sync response is cumulative and will contain
+                // the latest account data.
```

Comment:
I am somewhat surprised that `sendOrQueueSyncResponse` fails. why is that happening?

Reply:
It fails with `Cannot queue more than one /sync response` with current tests.
Which is not problematic in our case.

Reply:
> It fails with `Cannot queue more than one /sync response` with current tests.

hrm. could you record that in a comment?